spec/spec_helper.rb

```diff
@@ -23,6 +23,7 @@
 VCR.configure do |config|
   config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
   config.hook_into :webmock
+  config.filter_sensitive_data('<TOKEN>') { ENV.fetch('GITHUB_ACCESS_TOKEN') }
```

Comment:
This will prevent contributors from accidentally recording their access tokens in the cassettes 😱